src/agent/src/rpc.rs

```diff
@@ -1341,6 +1396,41 @@ fn get_memory_info(block_size: bool, hotplug: bool) -> Result<(u64, bool)> {
     Ok((size, plug))
 }
 
+fn get_volume_capacity_stats(path: &str) -> Result<VolumeUsage> {
+    let mut usage = VolumeUsage::new();
+
+    let s = System::new();
+    for disk in s.disks() {
+        if disk.name().to_str().unwrap().eq(path) {
+            usage.available = disk.available_space();
+            usage.total = disk.total_space();
+            usage.used = usage.total - usage.available;
+            usage.unit = VolumeUsage_Unit::BYTES; // bytes
+            break;
+        }
+    }
+
+    Ok(usage)
+}
+
+fn get_volume_inode_stats(path: &str) -> Result<VolumeUsage> {
+    let mut usage = VolumeUsage::new();
+
+    let s = System::new();
+    for disk in s.disks() {
+        if disk.name().to_str().unwrap().eq(path) {
+            let mount_point = disk.mount_point();
+            let meta = fs::metadata(mount_point.to_str().unwrap())?;
```

Comment:
nit: would be good to use `ok_or()` instead of `unwrap()` to convert `Option` to `Result` with descriptive error message.

Reply:
or even simpler
```rust
let meta = fs::metadata(disk.mount_point())?;
```